metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/controller/GenericEntitiesController.java

```diff
@@ -508,7 +508,7 @@ public ResponseEntity<E> createAspect(
       @PathVariable("aspectName") String aspectName,
       @RequestParam(value = "systemMetadata", required = false, defaultValue = "false")
           Boolean withSystemMetadata,
-      @RequestParam(value = "createIfNotExists", required = false, defaultValue = "false")
+      @RequestParam(value = "createIfNotExists", required = false, defaultValue = "true")
```

Comment:
This was recently added in https://github.com/datahub-project/datahub/pull/10654/ for us specifically and no one is using this afaik, but i will note it just in case

Reply:
Actually @RyanHolstien is that necessary if there was no release published between the changes?
This change (#10654) was implemented after v0.13.3 was released

Reply:
Nope! That's fine if it hasn't been released yet :)